compose/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Any modifications as to how this project can be built, since it got moved to the `compose` folder?